tests/plugins/test_tag_and_push.py

```diff
@@ -258,11 +269,34 @@ def custom_get(method, url, headers, **kwargs):
         if url == config_blob_url:
             return config_blob_response
 
+    def custom_get_v2missing(method, url, headers, **kwargs):
+        if url == manifest_latest_url:
+            # For a manifest stored as v2 or v1, the docker registry defaults to
+            # returning a v1 manifest if a v2 manifest is not explicitly requested
+            if headers['Accept'] == 'application/vnd.docker.distribution.manifest.v2+json':
+                return manifest_unknown_response
```

Comment:
`manifest_unknown_response` is the only value that changes between `custom_get` and `custom_get_v2missing`. Could we just edit `custom_get` to return `manifest_unknown_response` when needed if missing_v2==True ? This way we avoid the function duplication and some extra code below (line 292 to 299).

Reply:
yop will do that